packages/patternfly-4/react-charts/src/components/ChartLegend/examples/ChartLegend.md

```diff
@@ -475,7 +407,7 @@ class InteractiveLegendChart extends React.Component {
   }
 
   // Tips:
-  // 1. Don't omit components unless using custom colors -- that will reassign color scale
```

Comment:
Should we move these Tips from the example source code to the Tips section of the markdown documentation below, so they appear without having to expand the code block? I'm not sure, since we do still want people reading this example code to make sure they scroll down to see the tips. If we do move them, maybe we can leave a comment here like `// See the Tips section below for important notes about color scale, axis labels and tooltips`.

Reply:
Possibly. The tips at the bottom apply to all examples, while this applies only to this particular example

Reply:
Ah, I didn't realize it was specific to this example. I could go either way then.